etc/docker/.env.monitoring.dev

```
@@ -0,0 +1,26 @@
+DAPR_GRPC_PORT=42131
+DAPR_SECRET_STORE_NAME=vnext-secret
+DAPR_STATE_STORE_NAME=vnext-state
+DAPR_PUBSUB_STORE_NAME=vnext-pubsub
+DAPR_PUBSUB_BROADCAST_STORE_NAME=vnext-pubsub-broadcast
+DAPR_LOCK_STORE_NAME=vnext-lock
+OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector:4317
+OTEL_SERVICE_NAME=vnext-monitoring
+ConnectionStrings__Default=Host=vnext-postgres;Port=5432;Database=Aether_WorkflowDb;Username=postgres;Password=postgres;
+Redis__Standalone__EndPoints__0=vnext-redis:6379
+Vault__Enabled=false
+vNextApi__BaseUrl:http://vnext-app:5000
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)